instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/messaging/MessagingAttributesGetter.java

```diff
@@ -22,9 +22,6 @@
   @Nullable
   String getSystem(REQUEST request);
 
-  @Nullable
-  String getDestinationKind(REQUEST request);
```

Comment:
WDYT about adding a `default ... { return null; }` implementation for a deprecation cycle?